File: platforms/web/examples/scene/material/emissive_texture.js
import { Material, Renderer } from "fragmentcolor";

const renderer = new Renderer();
const glow_bytes = [ 255, 0, 0, 255, 255,   0, 0, 255, 255,   0, 0, 255, 255,   0, 0, 255, ];
const glow = await renderer.createTexture(glow_bytes, [2, 2]);
const mat = Material.pbr().emissive([0.8, 0.0, 0.0]).emissiveTexture(glow);

File: platforms/python/examples/scene/material/emissive_texture.py
from fragmentcolor import Material, Renderer

renderer = Renderer()
glow_bytes = [
    255, 0, 0, 255,
    255,   0, 0, 255,
    255,   0, 0, 255,
    255,   0, 0, 255,
]
glow = renderer.create_texture(glow_bytes, size=[2, 2])
mat = Material.pbr().emissive([0.8, 0.0, 0.0]).emissive_texture(glow)